national-resources/norway.md

```diff
@@ -0,0 +1,336 @@
+* [Norwegian Society for Medical Microbiology (Norsk forening for medisinsk mikrobiologi (NFMM)](https://www.legeforeningen.no/foreningsledd/fagmed/norsk-forening-for-medisinsk-mikrobiologi/) is a national association to bring together experts from medical microbiology fields for knowledge-exchange. NFMM also provides a [database of clinical laboratories that take care of infectious disease, pathogen, and related sampling](https://www.legeforeningen.no/foreningsledd/fagmed/norsk-forening-for-medisinsk-mikrobiologi/fag/kliniske-laboratorier-i-norge/).
+
+### Institutional initiatives on antimicrobial resistance (AMR)
+There are national research projects focused towards pathogenomics to help improve agriculture, food, public health, environment, and biodiversity in Norway. Also, higher education institutions and industrial organisations have forums and research centres dedicated to addressing the topics of antimicrobial resistance and pandemic preparedness.
+* [PathoSeq](https://nofima.com/projects/pathoseq/), a project on sequencing pathogenic bacteria for food safety. More information is available on the [University of Oslo PathoSeq page](https://www.jus.uio.no/ifp/english/research/projects/nrccl/pathoseq/).
+* [Interdisciplinary Forum for Research on Antimicrobial resistance (INFRA)](https://www.med.uio.no/she/english/network/INFRA/index.html), University of Oslo
+* [The Antibiotic Centre for Primary Care](https://www.med.uio.no/helsam/english/research/groups/general-practice-medicine/asp/), University of Oslo
+* The [Centre for new antibacterial strategies (CANS)](https://uit.no/research/cans?p_document_id=648381&Baseurl=/research/) at the Arctic University of Norway (UiT) comprises 16 research groups across three faculties, dedicated to AMR-related marine bioprospecting, pharmacology, and surveillance.
+
+### Institutional initiatives on epidemic/pandemic research
+* [Pandemic Centre](https://www.uib.no/en/pandemic) at the University of Bergen aims to address the topic of how to handle and prevent pandemics, and raise awareness through knowledge-exchange activities.
+* [Centre for Epidemic Interventions Research (CEIR)](https://www.fhi.no/en/about/departments-and-centres/ceir/), at the FHI, works towards understanding and further improving epidemic preparedness and risk control.
+
+In addition to national and municipality-level efforts, Norway has also been involved in international projects and initiatives to help address infectious disease control and contribute to research in the field .
+* [OneHealth](https://www.fhi.no/en/id/infectious-diseases/smitte-fra-mat-vann-og-dyr/flere-artikler/the-largest-ever-health-project-funded-from-the-eus-horizon-2020/)
```

Comment:
Perhaps these need a sentence to describe what each is in brief?

Reply:
Do you mean CEIR and OneHealth? For latter, the page has information about it in detail so to avoid repetition we didn't feel the need to add description.

Reply:
Do you mean CEIR and OneHealth? For latter, it is already mentioned under 'Projects' section of Disease surveillance initiatives: "The European Joint Programme One Health is a research program across Europe that promotes collaborative actions on foodborne zoonoses, antimicrobial resistance, and emerging threats."